config.yaml.sample

```diff
@@ -3,4 +3,6 @@ matrix:
   userid: "@sfu:shadowfax"
   accesstoken: "..."
 conference:
-  timeout: 30
+  pingPong:
+    timeout: 30
+    interval: 30
```

Comment:
Philosophical question: is there a more descriptive name that we could use for this? I know that keep-alive and heartbeat are used very often for the same meaning (the keep-alive is more used for marking connections alive, TCP's mechanism of keeping the connection alive is called like this; while heartbeat is oftentimes used for similar responsiveness checks).

[RFC 6455](https://www.rfc-editor.org/rfc/rfc6455#section-5.5.3) e.g. uses a heartbeat term for that.